core/src/components/item-sliding/test/standalone/e2e.ts

```diff
@@ -5,6 +5,45 @@ test('item-sliding: standalone', async () => {
     url: '/src/components/item-sliding/test/standalone?ionic:_testing=true'
   });
 
-  const compare = await page.compareScreenshot();
-  expect(compare).toMatchScreenshot();
+  const compares = [];
+
+  compares.push(await page.compareScreenshot());
+
+  // Grab item with start icons in option
+  let item = await page.find('#start');
+  await openItem(item, page);
+
+  compares.push(await page.compareScreenshot());
+
+  // Grab item with top icons in option
+  item = await page.find('#top');
+  await openItem(item, page);
+
+  compares.push(await page.compareScreenshot());
+
+  // Grab item with anchor option
+  item = await page.find('#anchor');
+  await openItem(item, page);
+
+  compares.push(await page.compareScreenshot());
+
+  for (const compare of compares) {
+    expect(compare).toMatchScreenshot();
+  }
 });
+
+async function openItem(item: any, page: any) {
```

Comment:
Can we move this to something a `test.utils.ts` file in `item-sliding/test`? 

That way we can have `openItem` and `deleteItem` functions and the tests in `tests/interactive` can call these functions as well.

Reply:
yeah sure if I can ever get screenshot to run :joy: